Source/WebCore/dom/EventTarget.cpp

```diff
@@ -88,6 +90,18 @@ bool EventTarget::addEventListener(const AtomString& eventType, Ref<EventListene
     listener->checkValidityForEventTarget(*this);
 #endif
 
+    if (RefPtr context = this->scriptExecutionContext()) {
+        if (context->requiresScriptExecutionTelemetry(ScriptTelemetryCategory::EventListeners)) {
+            if (eventType == eventNames().keydownEvent
```

Comment:
What about mouse/touch/pointer events?

Reply:
We monitor events that have the potential to capture or infer typed input.

Reply:
Have you tested what VoiceOver will do with voice input?

Reply:
I double-checked this with the Accessibility team, and their response was that VoiceOver relies on accessibility APIs, which web developers or telemetry scripts cannot directly access.